### PR TITLE
feat(Logout_confirmation): Add logout confirmation dialog

### PR DESCRIPTION
**Pull request by ssakone**

### CODE REVIEW

**Comment by smk762** [comment]
When attempting to log out via header bar, nothing happens.
When attempting to logout from settings, modal disappears, but app remains logged in.
No indication of cause in logs.

https://user-images.githubusercontent.com/35845239/121633153-dc11a500-cab4-11eb-8926-1f417f844ace.mp4

**Review by tonymorony — CHANGES_REQUESTED** [review]
@ssakone logout doesn't work at all on this branch, please re-check

**Comment by ssakone** [comment]
ok

**Comment by ssakone** [comment]
> When attempting to log out via header bar, nothing happens.
> When attempting to logout from settings, modal disappears, but app remains logged in.
> No indication of cause in logs.
> no_logout.mp4

pls tell me what os, because on linux i have this issue

**Comment by tonymorony** [comment]
The functionality works for me now, thanks!

@smk762 could you please re-test and advice on wording there:

image

File: atomic_defi_design/qml/Settings/SettingModal.qml [hunk 1]
```diff
@@ -18,9 +18,14 @@ import "../Constants"
 Qaterial.Dialog {
 
     function disconnect() {
-        app.currentWalletName = ""
-        API.app.disconnect()
-        onDisconnect()
+        
+        Qaterial.DialogManager.showDialog({title: qsTr("Logout request"),text: qsTr("Confirm logout request to disconnect!!"),iconSource: Qaterial.Icons.logout,standardButtons: Dialog.Yes | Dialog.Cancel, onAccepted: function(){
```
Comment (smk762) [inline atomic_defi_design/qml/Settings/SettingModal.qml:22]: Change title to "Confirm Logout"
Change text to "Are you sure you want to log out?"

**Review by tonymorony — APPROVED** [review]

**Pull request merged**